docs/x509/reference.rst

```diff
@@ -1860,6 +1860,40 @@ X.509 Extensions
 
         :type: int
 
+.. class:: PolicyConstraints
+
+    .. versionadded:: 1.3
+
+    The policy constraints extension can be used in certificates issued
+    to CAs.  The policy constraints extension constrains path validation
+    in two ways.  It can be used to prohibit policy mapping or require
+    that each certificate in a path contain an acceptable policy
+    identifier. For more information about the use of this extension see
```

Comment:
This is pretty hard to parse. Can you try simplifying? I can try to come up with a suggestion if it'd be helpful.